openfisca_core/entities.py

```diff
@@ -463,11 +513,25 @@ def value_from_person(self, array, role, default = 0):
 
         return result
 
-    def value_from_first_person(self, array):
+    def value_nth_person(self, n, array, default = 0):
+        """
+            Get the value of array for the nth persons of each entity.
+            If the nth person does not exist, get default instead
+            The result is a vector which dimension is the number of entities
```

Comment:
Proposition : 
> e.g.  household.value_nth_person(2, sorting_criteria) will return a vector with the 2nd ranking member of each household ranked according to the sorting_criteria.

Reply:
No, there is no sorting happening in this function. The "position" of a person in an entity is arbitrary. When you declare a family to OpenFisca, each person in the family is given a position (0 to k, where k+1 is the number of persons in the family).

This position has no meaning in the real world, so it is not usually not used in formulas. However, it can used for technical purpose, like in this PR.